src/pages/Preview/containers/Preview.js

```diff
@@ -2,15 +2,24 @@ import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import ReactPlayer from 'react-player';
 import { getSongByTitleId } from 'utils/ktvQueries';
+import { useGlobalContext } from '../../../global/context';
 
 const Preview = ({ match }) => {
   const {
     params: { songName },
   } = match;
+  const globalContext = useGlobalContext();
+  const [karaokeState, karaokeDispatch] = globalContext.karaoke;
 
   const [songTitle, setSongTitle] = useState('');
   const [artist, setArtist] = useState('');
-  const [play, setPlay] = useState(true);
+
+  const setPlaySong = (play) => {
+    karaokeDispatch({
+      type: 'SET_PLAYSONG',
+      payload: { playSong: play },
+    });
+  };
```

Comment:
Changing this to: ` const setPlaySong = useCallback((play) => {
      karaokeDispatch({
        type: 'SET_PLAYSONG',
        payload: { playSong: play },
      });
    },
    [karaokeDispatch],
  );`